pandas/io/tests/test_sql.py

```diff
@@ -1963,14 +1977,10 @@ def test_illegal_names(self):
         for ndx, ok_name in enumerate(['99beginswithnumber','12345']):
             df.to_sql(ok_name, self.conn, flavor=self.flavor, index=False,
                       if_exists='replace')
-            self.conn.cursor().execute("DROP TABLE `%s`" % ok_name)
-            self.conn.commit()
             df2 = DataFrame([[1, 2], [3, 4]], columns=['a', ok_name])
             c_tbl = 'test_ok_col_name%d'%ndx
```

Comment:
But I mean the variable `c_tbl` which is a string, and not used anymore? (the tables are indeed dropped in teardown)